src/neynar-api/v2/openapi-farcaster/models/ban-response.ts

```diff
@@ -0,0 +1,36 @@
+/* tslint:disable */
+/* eslint-disable */
+/**
+ * Farcaster API V2
+ * No description provided (generated by Openapi Generator https://github.com/openapitools/openapi-generator)
+ *
+ * The version of the OpenAPI document: 2.0
+ * 
+ *
+ * NOTE: This class is auto generated by OpenAPI Generator (https://openapi-generator.tech).
+ * https://openapi-generator.tech
+ * Do not edit the class manually.
+ */
+
+
+
+/**
+ * 
+ * @export
+ * @interface BanResponse
+ */
+export interface BanResponse {
+    /**
+     * 
+     * @type {boolean}
+     * @memberof BanResponse
+     */
+    'success': boolean;
+    /**
+     * 
+     * @type {string}
+     * @memberof BanResponse
+     */
+    'message'?: string;
```

Comment:
should be `string | null`

property can be set to null by adding `nullable: true`

<img width="498" alt="image" src="https://github.com/user-attachments/assets/5d699464-fc47-4170-be2f-dd4f9849a75c">

Or else why don't we use `OperationResponse` instead